src/components/common/Button.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const Button = ({ className, value, onClick }) => {
+  return (
+    <button className={className} value={value} onClick={onClick}>
+      {value}
+    </button>
+  );
+};
```

Comment:
```suggestion
const Button = ({ className, children, onClick }) => {
  return (
    <button className={className} onClick={onClick}>
      {children}
    </button>
  );
};
```

이런 완전 도메인과 비종속적인 가장 작은 단위의 컴포넌트들은 HTML 태그와 거의 유사한 레벨에서 사용되기 때문에,
HTML태그와 인터페이스를 맞추는게 좋습니다.
그래야 이 컴포넌트들을 사용하는 개발자들이 더 빠르고 쉽게 사용할 수 있게 됩니다 😄

Reply:
좋은 말씀 감사합니다!

위와 같은 HTML과 유사한 컴포넌트를 구현할 땐 말씀 주신 부분을 유의해서 작업하도록 하겠습니다!